From c793644518fc3ac982c3afb9cc2ceaed2d71adce Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Tue, 7 Dec 2021 15:08:30 +0000
Subject: [PATCH] configs: rzg2l: Modify environment to work with the Barcode
 PoC

The Barcode PoC uses the same U-Boot machine as the SMARC RZ/G2L
board, however the SMARC RZ/G2L machine has 2 SD cards and the
environment is defaulting to using the 2nd SD card, whereas in
the Barcode PoC case we need to use the SD found on the SoM.
The changes from this commit allow for using the SD card found
on the RZ/G2L SMARC SoM.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 include/configs/smarc-rzg2l.h | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/include/configs/smarc-rzg2l.h b/include/configs/smarc-rzg2l.h
index f543b34f40..e1a839b249 100644
--- a/include/configs/smarc-rzg2l.h
+++ b/include/configs/smarc-rzg2l.h
@@ -61,12 +61,10 @@
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"bootm_size=0x10000000 \0" \
-	"prodsdbootargs=setenv bootargs rw rootwait earlycon root=/dev/mmcblk1p1 \0" \
-	"prodemmcbootargs=setenv bootargs rw rootwait earlycon root=/dev/mmcblk0p1 \0" \
+	"prodsdbootargs=setenv bootargs rw rootwait earlycon root=/dev/mmcblk0p1 \0" \
 	"bootimage=unzip 0x4A080000 0x48080000; booti 0x48080000 - 0x48000000 \0" \
-	"emmcload=ext4load mmc 0:1 0x4A080000 boot/Image.gz;ext4load mmc 0:1 0x48000000 boot/r9a07g044l-smarc-rzg2l.dtb;run prodemmcbootargs \0" \
-	"sd1load=ext4load mmc 1:1 0x4A080000 boot/Image.gz;ext4load mmc 1:1 0x48000000 boot/r9a07g044l-smarc-rzg2l.dtb;run prodsdbootargs \0" \
-	"bootcmd_check=if mmc dev 1; then run sd1load; else run emmcload; fi \0"
+	"sd0load=ext4load mmc 0:1 0x4A080000 boot/Image.gz;ext4load mmc 0:1 0x48000000 boot/r9a07g044l2-barcode-poc.dtb;run prodsdbootargs \0" \
+	"bootcmd_check=if mmc dev 0; then run sd0load; fi \0"
 
 #define CONFIG_BOOTCOMMAND	"env default -a;run bootcmd_check;run bootimage"
 
-- 
2.25.1

